doc/api/async_hooks.md

```diff
@@ -712,6 +712,75 @@ never be called.
 * Returns: {number} The same `triggerAsyncId` that is passed to the
 `AsyncResource` constructor.
 
+## Class: AsyncStorage
+
+This class is used to create contexts that can be used through the event loop.
```

Comment:
This really isn't that clear. An example here would be helpful.

Reply:
Moving the example below up would be sufficient.

Reply:
Good point. will do.